pkg/storage/stores/shipper/util/util.go

```diff
@@ -72,11 +73,13 @@ type IndexStorageClient interface {
 
 // GetFileFromStorage downloads a file from storage to given location.
 func GetFileFromStorage(ctx context.Context, storageClient IndexStorageClient, tableName, fileName, destination string, sync bool) error {
+	start := time.Now()
 	readCloser, err := storageClient.GetFile(ctx, tableName, fileName)
 	if err != nil {
 		return err
 	}
 
+	downloaded := time.Now().Sub(start)
```

Comment:
The `GetFile` actually returns a reader to the file, which would not include download latency.
It would be hard to calculate it since we decompress the file as we download it by chaining the readers.

Reply:
great catch sandeep, I simplified to just log the total time